src/config/webpack/webpack.dev.config.js

```diff
@@ -0,0 +1,38 @@
+const webpack = require("webpack")
+const path = require("path")
+const merge = require("webpack-merge")
+
+module.exports = () => merge([
```

Comment:
I think, using 'merge' here is unnecessary. You return only one definition.

Reply:
ok